actors/miner/tests/miner_actor_test_construction.rs

```diff
@@ -100,10 +145,21 @@ fn simple_construction() {
     assert_eq!(2349, info.window_post_partition_sectors);
 
     assert_eq!(TokenAmount::zero(), state.pre_commit_deposits);
-    assert_eq!(TokenAmount::zero(), state.locked_funds);
+    assert_eq!(TokenAmount::from_atto(633318697598976000u64), state.locked_funds);
```

Comment:
ditto here, it would be good to involve some constants or named variables to at least describe the starting point and the calculation used to arive here